docs/map-making/how-to-upgrade-legacy-map-features.md

```diff
@@ -41,43 +32,20 @@ Within every game (XML) file, inside the "games" folder inside the main folder o
 - `="Battleships repair at beginning of round"` >> `="Units Repair Hits Start Turn"`
 - `="Naval Bombard Casualties Return Fire Restricted"` >> `="Naval Bombard Casualties Return Fire"`
 - `="SBR Affects Unit Production"` >> `="Damage From Bombing Done To Units Instead Of Territories"`
-
-### Changes of game (XML) file special code for the above items:
```

Comment:
The reason why I created two "special code" sections is that such changes touch items that are very rarely found in games. Practically, only games having triggers changing values of options may need making such changes. Maybe the "so you probably don't have these within the game file" note could have been clarified by explaining that games with no triggers don't need having these changes applied to them (even though games with triggers are unlikely to have them too). Consolidating all in a single section is a reasonable preference, but this means having a lot of changes code that is very rarely useful, thus my preference of splitting between common and uncommon changes to make.

Reply:
The distinction is really not that important and adds a lot of verbiage for the sake of "yeah, you have to change these too, but you probably don't have them.". It's not that important of a distinction. What we're looking for in this list is a s simple as possible,  "change this, into that".